### ISSUE TITLE
The GDB help is broken for commands

### ISSUE THREAD

**Issue by disconnect3d** [pwndbg/pwndbg#2350]
@mbrla0 I believe the debugger-agnostic command API broke the `help <command>` available from GDB:

```
pwndbg> help nearpc
This command is not documented.
```

The `--help` from argparse works though:
```
pwndbg> nearpc --help
usage: nearpc [-h] [-e] [pc] [lines]

Disassemble near a specified address.

positional arguments:
  pc             Address to disassemble near. If this is the only argument and the
                 value provided is small enough, it is interpreted as lines instead.
  lines          Number of lines to show on either side of the address.

options:
  -h, --help     show this help message and exit
  -e, --emulate  Whether to emulate instructions to find the next ones or just linearly
                 disassemble. (default: False)
```

But yeah, we do want to register the help strings into GDB interface.

**Comment by mbrla0** [comment]
I failed to realize GDB used the _docstring_ of all things to determine the help string. #2351 should fix it.

**Comment by disconnect3d** [comment]
Fixed in #2351
